apps/protocol-frontend/src/components/EditContributionForm.tsx

```diff
@@ -76,15 +77,15 @@ const EditContributionForm = ({ contribution }: EditContributionFormProps) => {
     userId: userData?.id,
   });
 
-  const daoListOptions =
+  const daoListOptions: Option<number>[] =
     useUserData?.guild_users.map(dao => ({
       value: dao.guild.id,
       label: dao.guild.name ?? '',
     })) || [];
 
-  const daoReset = [
+  const daoReset: Option<number>[] = [
     {
-      value: null,
+      value: -1,
```

Comment:
It looks like this change is breaking the mutation -- we'll likely need to update the resolver. Changing this back to `null` resolves the error I mentioned in another comment

Reply:
The resolver has a check for null -- if we want to use `-1` instead we'll want to modify the contribution update resolver. Or we can modify the `SelectOption` type to allow for `null` and keep this as `null` on the reset.

@alexkeating what do you think?